it/http/src/test/java/com/adobe/cq/commerce/it/http/CommerceTestBase.java

```diff
@@ -185,11 +190,16 @@ protected Elements getItems(Element e, String itemTypeRegex) {
      * @return
      */
     protected String getValue(Element e) {
-        if (e.tagName().equals("meta")) return e.attr("content");
-        if (e.tagName().equals("a")) return e.attr("href");
-        if (e.tagName().equals("link")) return e.attr("href");
-        if (e.tagName().equals("img")) return e.attr("src");
-        if (e.tagName().equals("input")) return e.attr("value");
+        if (e.tagName().equals("meta"))
```

Comment:
The AEM formatter should enforce brackets even for single line blocks. Let me check if I can configure this in the formatter config and I'll give you another version.

Reply:
Unfortunately, this is configured elsewhere (that is, not in the formatter rules) in Eclipse, and I realised it's the same for the order of the import statements. Can you check how/if this can be anyway configured in the formatter plugin?

Reply:
Sorry, I should have checked this myself: https://github.com/revelc/impsort-maven-plugin

Reply:
@jelger I checked at https://code.revelc.net/formatter-maven-plugin/examples.html#Custom_Configuration_File. Seems like the formatter only supports configurations from eclipse compatible formatting files, but nothing else :/

Reply:
@cjelger Looks good, I'll add this plugin to the PR.